docs/UserGuide.adoc

```diff
@@ -308,20 +308,16 @@ Lists all patients only.
 Edits the information of the person with the specified NRIC to the attributes specified.
 It is up to the user to specify which attributes should be updated. +
 
-Format: `edit ic/**NRIC**`
```

Comment:
we changed it to not require `ic/` anymore?

Reply:
Yes